src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const findMen = (man) => {
+    return (century === undefined) ? (man['sex'] === 'm') : (
```

Comment:
Check your code style and fix everywhere according this comment
```suggestion
    return !century ? (man.sex === 'm') : (
```